### PR TITLE
bpfman release v0.5.4

### PR DESCRIPTION
**Pull request by anfredette**

### CODE REVIEW

**Review by msherif1234 — COMMENTED** [review]
/LGTM

File: changelogs/CHANGELOG-v0.5.4.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+The v0.5.4 release is a patch release that fixes a libc incompatibility between
```
Comment (Billy99) [inline changelogs/CHANGELOG-v0.5.4.md:1]:
```suggestion
The v0.5.4 release is a patch release that fixes a glibc incompatibility between
```

**Review by Billy99 — COMMENTED** [review]
nit
Also, do we want to update the v0.5.2 and v0.5.3 release notes with a statement on why not to use them?

File: changelogs/CHANGELOG-v0.5.4.md (hunk 1 above)
Comment (anfredette) [reply changelogs/CHANGELOG-v0.5.4.md:1]: >Also, do we want to update the v0.5.2 and v0.5.3 release notes with a statement on why not to use them?

I'll give it a try.

Comment (anfredette) [reply changelogs/CHANGELOG-v0.5.4.md]: glibc change made.  I also added a "Known Issues" section for v0.5.2 and v0.5.3.

**Review by Billy99 — APPROVED** [review]
/lgtm

**Pull request merged**